src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,137 @@
+p,
+h2 {
+  margin-block-start: 0;
+  margin-block-end: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  font-family: 'Roboto', sans-serif;
+  width: 200px;
+  height: 406px;
+  border-radius: 5px 0px 0px 0px;
+  border: 1px solid #f3f3f3;
+  opacity: 0px;
+
+  padding: 32px 16px 16px;
+  margin: 0;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+
+    margin: 0 auto 40px;
+    // justify-content: center;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+
+    margin-bottom: 4px;
+
+    color: #060b35;
+  }
+
+  &__code {
+    display: block;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+
+    color: #616070;
+
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+
+    margin-bottom: 24px;
+
+    &-amount {
+      display: block;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+
+      color: #060b35;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin-bottom: 16px;
+
+    &-title {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+
+      color: #616070;
+    }
+
+    &-amount {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+
+      color: #060b35;
+    }
+  }
+
+  &__buy {
+    display: flex;
+    justify-content: center;
+    text-align: center;
+    box-sizing: border-box;
+    padding-block: 12px;
+    height: 40px;
+    border: 1px solid #00acdc;
+    border-radius: 5px;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: #ffffff;
+    background-color: #00acdc;
+
+    &:hover {
+      color: #00acdc;
+      background-color: #ffffff;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
```

Comment:
Using `:last-child` is a good way to ensure that the last element doesn't have a right margin. Good job on that. Just verify that the `4px` margin for the other stars aligns with the design requirements.